packages/slice-machine/tests/project/cypress/integration/customTypes/00-create.spec.js

```diff
@@ -14,11 +16,14 @@ describe("Custom Types specs", () => {
       .get('[data-cy=create-ct-modal]')
       .should('be.visible');
 
-    cy.get('input[data-cy=ct-name-input]').type('My custom type')
-    cy.get('input[data-cy=ct-id-input]').type('my-custom-type')
+    const name = [randomString(), randomString()].map(capitalizeFirstChar).join(" ")
```

Comment:
hum I'm not comfortable with using random stuff in test, that not easily reproductible if there is a fail also that doesn't ensure toi test properly the case we want to cover.

Reply:
I get ffile conflicts when running the test with the same name, it's a pain to keep having to delete files.